int8 count
int8 legs_number=4
geometry_msgs/Point[] legs
geometry_msgs/Point[] center